instagram-edge-app/instagram/static/js/main.js

```diff
@@ -0,0 +1,47 @@
+(function () {
+  function displayPosts (posts) {
+    const container = document.querySelector('.cards')
+    const placeholder = document.querySelector('.dummy-card')
+    const fragment = document.createDocumentFragment()
+
+    posts.forEach(({ username, caption, media_url: mediaUrl }) => {
+      const node = placeholder.cloneNode(true)
+      const imgNode = node.querySelector('.post-image')
+      const captionNode = node.querySelector('.caption')
+
+      imgNode.setAttribute('src', mediaUrl)
+      captionNode.innerHTML = `<span class="user-handle">@${username}</span> ${caption || ''}`
+      node.classList.remove('dummy-card')
+      fragment.appendChild(node)
+    })
+
+    container.appendChild(fragment)
+  }
+
+  function fetchPosts () {
+    const token = screenly.settings.instagram_api_token
+    // const token = "IGQWRQbXE5YmJuWFI5RjRYQkxYdkJlYTZA1ZAHhELS1JUkZAyS05wNlJQcXY0QlFjODFYWGVIY0JLQ2RGZAlJBOWdyaUQ0bnJzMkwwUXVFd0tVYzhmaXVrRVdTcTEtTVBYendob1VfYWZAPTXRsRTlxQUZAvZAWQ1b1RfM0UZD"
```

Comment:
Please don't leave in a hard coded token (and make sure to revoke this token asap).

Reply:
Understood, token has already been revoked. I am sending a new PR with the updated files

Reply:
No no. Don't send a new PR. Just update this one.